dist/tools/pic32prog/doc.md

```diff
@@ -0,0 +1,160 @@
+Flashing on Linux using pickit-3
+================================
+
+This document describes how to enable flashing a pic32-wifire from Linux with a
+PICkit3 using [pic32prog](https://github.com/sergev/pic32prog).
+
+It will require flashing a specific firmware on the PICkit3.
+As this can only be done from a Windows computer, that not many Linux users
+have, the following steps explain how to setup a Windows VirtualBox virtual
```

Comment:
This document contains several spellings of `VirtualBox` (capitalization differs). Please decide on one (I prefer the canonical spelling).

Reply:
Updated all to `VirtualBox`.